js/dataframe/shared/Table.svelte

```diff
@@ -152,7 +160,8 @@
 	$: _headers &&
 		dispatch("change", {
 			data: data.map((r) => r.map(({ value }) => value)),
-			headers: _headers.map((h) => h.value)
+			headers: _headers.map((h) => h.value),
+			metadata: editable ? null : { display_value: display_value }
```

Comment:
Originally just had:

> metadata: {display_value: display_value }

but I'd get an infinite loop in the InteractiveDataframe. I think I'm doing something wrong here

Reply:
I'm entirely sure why this would be happening. I know that the dataframe does update more than it should and can be very sensitive to changes triggering new updates but I don't really understand why this would stop the update from happening because this branch trigger an update to the value regardless. I'll have a poke around and see if I can figure out what might be causing it.

I'll probably take a look at the way we handle state in the dataframe shortly and do sonmething more explicit than what we currently have. For now I don't want to block this. I can see any actuall issues, so if my poking and prodding doesn't turn up anything I think it is fine to merge until we refine how we handle state and updates in general for the DF.